build.gradle

```diff
@@ -1,7 +1,9 @@
 buildscript {
     repositories {
         gradlePluginPortal()
```

Comment:
I think we may want the same thing here, the gradle plugin portal redirects to other repositories (thus the pain when jcenter goes down)
```suggestion
        gradlePluginPortal() {
            metadataSources { mavenPom(); ignoreGradleMetadataRedirection() }
        }
```

Reply:
Probably worth excavating this out to our oss projects